lib/eventFields.js

```diff
@@ -91,6 +92,16 @@ export const EventType = new GraphQLObjectType({
     },
     datetime: {
       type: EventDateTimeType,
+      description: '[DEPRECATED] Date and time of the event',
+      deprecationReason: 'Renamed to "startDateTime"',
```

Comment:
Good catch. I think it might be better to explicitly specify the fields for news though, instead of directly using the EventType. I can pull them out onto a "commonFields" module or something, perhaps?